hack/update-bazel.sh

```diff
@@ -104,3 +103,6 @@ buildozer -quiet 'set clinkopts select({"@io_bazel_rules_go//go/platform:linux":
 if [[ $ret != 0 && $ret != 3 ]]; then
   exit 1
 fi
+
+# Avoid bazel stuff in tools/ directory
+rm hack/tools/BUILD
```

Comment:
is this still needed with the exclusion of hack/tools in `build/root/.kazelcfg.json`?

Reply:
yes, apparently the exclusion is not enough.